Update the user management documentation (#683)
owncloud/docs-server: modules/admin_manual/pages/configuration/user/user_management.adoc

{description}

> Note that most user related tasks can also be managed via xref:configuration/server/occ_command.adoc#user-commands[occ commands for managing users].

Reviewer: ```suggestion
Note that most user-related tasks can also be managed via xref:configuration/server/occ_command.adoc#user-commands[occ commands for managing users].
```

**User accounts** have the following **properties**:

_Login Name (Username)_::

>   The unique ID of an ownCloud user. Note that it **cannot** be changed post creating the user. The user can login either using the login name or the E-mail address. 

Reviewer: ```suggestion
  The unique ID of an ownCloud user. Note that it **cannot** be changed after creating the user. The user can log in either using the login name or the e-mail address. 
```

_Full Name_::

>   The user’s display name that appears on file shares, the ownCloud Web interface, and E-mails. Admins and users may change the Full Name anytime. If the Full Name is not set, it defaults to the login name.

Reviewer: ```suggestion
  The user’s display name that appears on file shares, the ownCloud Web interface and emails. Admins and users may change the full name anytime. If the full name is not set, it defaults to the login name.
```

_Password_::
  The admin sets the new user’s first password. Both, the user and the admin can change the user’s password at anytime.
_E-Mail_::

>   The admin sets the new user’s E-Mail. The user then gets an E-Mail to set the password. Both, the user and the admin can change the user’s E-Mail at anytime.

Reviewer: ```suggestion
  The admin sets the new user’s e-mail. The user then gets an e-mail to set the password. Both, the user and the admin can change the user’s e-mail at anytime.
```

_Groups_::
  You may create system (non-LDAP) groups and assign system group memberships to users. By default, new users are not assigned to any groups.
_Group Admin_::

>   Group admins are granted administrative privileges on system groups, and can add new users and remove existing users from those groups.

Reviewer: ```suggestion
  Group admins are granted administrative privileges on system groups and can add new users and remove existing users from those groups.
```

image::configuration/user/delete-user-confirmation.png[The ownCloud delete user confirmation dialog]

> To delete a user, hover your cursor over their name on the *Users* page, and click the trashcan icon that appears at the far right. You’ll then see a confirmation dialog appear, asking if you’re sure that you want to delete the user. 

Reviewer: ```suggestion
To delete a user, hover your cursor over their name on the *Users* page and click the trashcan icon that appears at the far right. You’ll then see a confirmation dialog appear, asking if you’re sure that you want to delete the user. 
```

If you click btn:[Yes], the user is permanently deleted, including all of the files owned by the user, including all files they have shared. If you need to preserve the user’s files and shares, you must first download them from your ownCloud Files page, (which compresses them into a zip file). 

Alternatively, you can use a sync client to copy them to your local computer. If you click btn:[No], the confirmation dialog will disappear and the user is not deleted.

TIP: See xref:configuration/files/file_sharing_configuration.adoc[File Sharing Configuration] to learn how to create persistent file shares that survive user deletions.

== Granting Administrator Privileges

ownCloud has two types of administrators: 

> * *ownCloud Administrators* have full rights on your ownCloud server, and can access and modify all settings. To assign the ownCloud Administrators role to a user, simply add them to the `admin` group.

Reviewer: ```suggestion
* *ownCloud Administrators* have full rights on your ownCloud server and can access and modify all settings. To assign the ownCloud administrator role to a user, simply add them to the `admin` group.
```

=== Quota Field

Found in menu:User Authentication[the Advanced Tab > Special Attributes], this setting overwrites the rest. If set, this is what will be set for an LDAP user’s quota in ownCloud.

=== Quota Default

Found in menu:User Authentication[the Advanced Tab > Special Attributes], this is the fallback option if no quota field is defined.

=== User Quota

This is what you set in the web UI drop down menu, and is how you set user quota.

=== Default Quota

> This will be set if no quota is set, and is found in menu:Users Tab[Gear Wheel > Default Quota]. If the *Quota Field* is not set, but *Quota Default* is, and a systems administrator tries to set a quota for an LDAP user with *User Quota*, it will not work, since it is overridden by *Quota Default*.

Reviewer: ```suggestion
This will be used if no quota is set and is found in menu:Users Tab[Gear Wheel > Default Quota]. If the *Quota* is not set, but *Default Quota* is, and a systems administrator tries to set a quota for an LDAP user with *User Quota*, it will not work since it is overridden by *Default Quota*.
```

> Click the btn:[gear] icon on the lower left pane to set a default storage quota. This is automatically applied to new users. You may assign a different quota to any user by selecting from the *Quota* dropdown, selecting either a preset value or entering a custom value. When you create custom quotas, use the normal abbreviations for your storage values such as 500 MB, 5 GB, 5 TB, and so on.

Reviewer: ```suggestion
Click the btn:[gear] icon on the lower left pane to set a default storage quota. This is automatically applied to new users. You may assign a different quota to any user by selecting a preset value from the *Quota* dropdown or by entering a custom value. When you create custom quotas, use the usual abbreviations for your storage values such as 500 MB, 5 GB, 5 TB, and so on.
```

=== External Storage Quota

> You now have a configurable option in `config.php` that controls whether external storage is counted against user’s quotas. This is still experimental, and may not work as expected. The default is to not count external storage as part of user storage quotas. If you prefer to include it, then change the default `false` to `true`.:

Reviewer: ```suggestion
You now have a configurable option in `config.php` that controls whether external storage is counted against user’s quotas. This is still experimental and may not work as expected. The default is to not counting external storage as part of user storage quotas. If you prefer to include it, then change the default `false` to `true`.:
```

[source,php]
----
'quota_include_external_storage' => false,
----

=== Storage Space Considerations

> Metadata (such as thumbnails, temporary files, and encryption keys) takes up about 10% of disk space, but is not counted against user quotas. Users can check their used and available space on their Personal pages. Only files that originate with users count against their quotas, and not files shared with them that originate from other users. For example, if you upload files to a different user’s share, those files count against your quota. If you re-share a file that another user shared with you, that file does not count against your quota, but the originating user’s.

Reviewer: ```suggestion
Metadata (such as thumbnails, temporary files, and encryption keys) takes up about 10% of disk space but is not counted against user quotas. Users can check their used and available space on their Personal pages. Only files that originate from a user count against that user's quota, not files shared by other users. For example, if you upload files to a different user’s share, those files count against your quota. If you re-share a file that another user shared with you, that file does not count against your quota, but the originating user’s.
```

Encrypted files are a little larger than unencrypted files; the unencrypted size is calculated against the user’s quota.

> Deleted files that are still in the trash bin do not count against quotas. The trash bin is set at 50% of quota. Deleted file aging is set at 30 days. When deleted files exceed 50% of quota then the oldest files are removed until the total is below 50%.

Reviewer: ```suggestion
Deleted files that are still in the trash bin do not count against quotas. The trash bin is set to 50% of quota. Deleted file aging is set to 30 days. When deleted files exceed 50% of quota, then the oldest files are removed until the total is below 50%.
```

=== Versions

> When version control is enabled, the older file versions are not counted against quotas.

Reviewer: ```suggestion
If version control is enabled, the older file versions are not counted against quotas.
```
